docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+        python manage.py migrate &&
+        python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:14-alpine
+    restart: always
+    ports:
+      - "5433:5432"
+    env_file:
+      - .env
+    volumes:
+      - my_db:/var/lib/postgresql/data
+
+volumes:
+  my_db: 
+    driver: local 
+    driver_opts:
+      type: none
+      device: C:/Users/Dennie/Postgres-data/

```

Comment:
The `device` path for the `my_db` volume is set to a specific Windows path (`C:/Users/Dennie/Postgres-data/`). Ensure that this path is correct and accessible from the Docker environment, especially if deploying on a different system or environment.